websites/N/Nexon Logistics/metadata.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "Nexon Logistics",
+  "author": {
+    "id": "757284167972159489",
+    "name": "FaizurRahman#5151"
```

Discriminator added by mistake by pmd
```suggestion
    "name": "FaizurRahman"
```